docs/install.md

```diff
@@ -1485,6 +1485,19 @@ deployment is in the `Failed` state, then one can look into the event(s) using
 > driver could run can be configured by using `nodeSelector` property of
 > [`DeploymentSpec`](#deployment-crd-api).
 
+#### Operator metrics data
+
+PMEM-CSI operator exposes below metrics data about active PmemCSIDeployment
+custom resources and it's sub-object in addition to the
+[metrics data provided by the controller-runtime](https://book-v1.book.kubebuilder.io/beyond_basics/controller_metrics.html):
+
+Name | Type | Explanation
+-----|------|------------
+`pmem_csi_deployment_reconcile_start` | counter | Counter that gets incremented on each time a PmemCSIDeployment CR enters a reconcile loop, labeled with the deployment name.
+`pmem_csi_deployment_reconcile_end` | counter | Counter that gets incremented on end of each reconcile loop of a PmemCSIDeployment CR, labeled with the deployment name.
```

Comment:
How useful is it to count function entry and exit?

What I had suggested on Monday was to count reconciliation, i.e. have a single `pmem_csi_deployment_reconcile` counter.